docker/apache/vhost.conf.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+cat << EOF > /usr/local/apache2/conf.d/000_${SORMAS_SERVER_URL}.conf
+<VirtualHost *:80>
+    ServerName ${SORMAS_SERVER_URL}
+    <IfModule mod_rewrite.c>
+    RewriteEngine On
+    RewriteCond %{HTTPS} !=on
+    RewriteRule ^/(.*) https://${SORMAS_SERVER_URL}/$REQUEST_URI [R,L]
+   </IfModule>
+   <IfModule !mod_rewrite.c>
+   	Redirect 301 /$REQUEST_URI https://${SORMAS_SERVER_URL}/$REQUEST_URI
+   </IfModule>
+</VirtualHost>
+EOF
+
+cat << EOF > /usr/local/apache2/conf.d/001_ssl_${SORMAS_SERVER_URL}.conf
+Listen 443
+<VirtualHost *:443>
+        ServerName ${SORMAS_SERVER_URL}
+
+        Header set Access-Control-Allow-Origin "*"
```

Comment:
This will forbid credentialed requests (HTTP cookies and HTTP Authentication information). I'm not sure this is desired. See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#requests_with_credentials) subsection "Credentialed requests and wildcards".

Reply:
Yep, I think also `Access-Control-Allow-Credentials: true` needs to be also set. This method now allows communication from Angular client to API on the `https://sormas-docker-test.com` domain - because this doesn't depend on cookies. 

But connection from Keycloak to Angular uses cookies.